### PR TITLE
replace helm with kapp in 0.5.0 getting started

### LINKED ISSUE #169: Switch getting started guides to use kapp
Change the helm commands to use kapp equivalents

### PR DESCRIPTION
**Pull request by ekcasey**
Fixes #169

### CODE REVIEW

File: docs/v0.5/getting-started/docker-for-mac.md [hunk 1]
```diff
@@ -82,36 +102,55 @@ riff --version
...
+```
+kubectl create ns apps
+```
+
+### install riff Build
+To install riff build and it's dependencies:
 ```sh
-helm repo add projectriff https://projectriff.storage.googleapis.com/charts/releases
-helm repo update
+   kapp deploy -n apps -a cert-manager -f https://storage.googleapis.com/projectriff/charts/uncharted/0.5.0-snapshot/cert-manager.yaml
```
Comment (trisberg) [inline docs/v0.5/getting-started/docker-for-mac.md]: the extra indents for these commands aren't needed

File: docs/v0.5/getting-started/docker-for-windows.md [hunk 2]
```diff
@@ -112,42 +122,55 @@ riff --version
...
+Create a namespace for kapp to store configuration:
+```
+kubectl create ns apps
 ```
 
-## Install riff using Helm
+### install riff Build
+To install riff build and it's dependencies:
+```sh
+   kapp deploy -n apps -a cert-manager -f https://storage.googleapis.com/projectriff/charts/uncharted/0.5.0-snapshot/cert-manager.yaml
```
Comment (trisberg) [inline docs/v0.5/getting-started/docker-for-windows.md]: some extra indents here too

File: docs/v0.5/getting-started/gke.md [hunk 3]
```diff
@@ -165,36 +163,54 @@ riff --version
...
+```
+kubectl create ns apps
+```
 
+### install riff Build
+To install riff build and it's dependencies:
 ```sh
-helm repo add projectriff https://projectriff.storage.googleapis.com/charts/releases
-helm repo update
+   kapp deploy -n apps -a cert-manager -f https://storage.googleapis.com/projectriff/charts/uncharted/0.5.0-snapshot/cert-manager.yaml
```
Comment (trisberg) [inline docs/v0.5/getting-started/gke.md]: extra indents ...

File: docs/v0.5/getting-started/minikube.md [hunk 4]
```diff
@@ -99,36 +119,55 @@ riff --version
...
+```
+kubectl create ns apps
+```
+
+### install riff Build
+To install riff build and it's dependencies:
 ```sh
-helm repo add projectriff https://projectriff.storage.googleapis.com/charts/releases
-helm repo update
+   kapp deploy -n apps -a cert-manager -f https://storage.googleapis.com/projectriff/charts/uncharted/0.5.0-snapshot/cert-manager.yaml
```
Comment (trisberg) [inline docs/v0.5/getting-started/minikube.md]: extra indents ...

**Review by trisberg — CHANGES_REQUESTED** [review]
LGTM except for the indents I commented on that should be removed

Two points we might want to consider:

1. Should we include uninstall instructions for the previous version, seems a bit heavy handed to have to delete the environment and recreate it. There are some notes here: https://github.com/projectriff/charts/tree/v0.4.0#uninstall
2. For Docker for Desktop I typically install the primary ingress controller with `LoadBalancer` since that seems to work fine and the service can be accessed on localhost at port 80 and I can add entries in `/etc/hosts` for my deployed knative services. This only works for a single `LoadBalancer` though, so not sure if we want to rely ion that for these instructions. Thoughts?

**Review by trisberg — APPROVED** [review]
LGTM, I'll open new issues for my 2 earlier questions

File: docs/v0.5/getting-started/docker-for-mac.md [hunk 5]
```diff
@@ -42,31 +42,51 @@ Watch pods in a separate terminal.
...
+
+[ytt](https://get-ytt.io/) is a tool for templating yaml. It can be used to apply changes to the distributed Kubernetes yamls files used to install riff.
+
+You install ytt using Homebrew:
+```sh
+brew tap k14s/tap
+brew install ytt
+```
+
+Alternatively, Download a recent binary for your platform from [github]((https://github.com/k14s/ytt/releases)).
```
Comment (jldec) [inline docs/v0.5/getting-started/docker-for-mac.md]: too many parentheses in the github link
Screen Shot 2020-01-13 at 5 41 05 PM

**Pull request merged**